inc/helpers/namespace.php

```diff
@@ -49,6 +49,7 @@ function toc_sections( $sections, $post_type, $can_read, $can_read_private, $per
 						$id,
 						$name
 					);
+					$subsection_output = strip_shortcodes( $subsection_output );
```

Comment:
This works, but I would prefer: 

```php
$subsection_output .= sprintf(
	'<li class="toc__subsection"><a href="%1$s#%2$s">%3$s</a></li>',
	get_permalink( $section['ID'] ),
	$id,
	strip_shortcodes( $name )
);
```

The targeting of `$name` (instead of `<ol class="toc__subsections"><li class="toc__subsection"><a href="someId">{$name}` would be more implicit.